README.md

```diff
@@ -43,6 +43,13 @@ yarn ts-node ./scripts/fetch-referrals.ts --protocol Beefy --output output.csv
 yarn ts-node ./scripts/referrer-user-count.ts --protocol Beefy --referrerIds app1,app2,app3 --networkIds celo-mainnet,base-mainnet
 ```
 
+### Fetch Revenue Summary
```

Comment:
Add a short summary describing what this script actually does/the data is returns?